guides/common/modules/ref_overview-of-the-host-columns.adoc

```diff
@@ -0,0 +1,44 @@
+[id="overview-of-the-host-columns_{context}"]
+= Overview of the Host Columns
+
+Below is the complete overview of columns that can be displayed in the host table divided into content categories.
+
+General::
+* Power - current state of the host, on/off
+* Name - name of the host
+* Operating system - operating system of the host
+* Owner - user owning the host
+* Host group - host group of the host
+* Boot time - last boot time of the host
+* Last report - time of the last host report
+* Comment - comment given to host
+
+Content::
+* Name - name of the host
+* Operating system - operating system of the host
+* Subscription status - does the host have a valid subscription attached.
+* Installable updates - numbers of installable updates divided into four categories: security, bugfix, enhancement, total
+* Lifecycle Environment - lifecycle environment of the host
+* Content view - content view of the host
+* Registered - time when the host was registered to satellite.
+* Last checkin - last time of the communication between the host and the Satellite server
+
+Network::
+* IPv4 - IPv4 address of the host
+* IPv6 - IPv6 address of the host
+* MAC - MAC address of the host
+
+Reported data::
+* Model - host hardware model
+* Sockets - number of host sockets
+* Cores - number of host processor cores
+* RAM - host RAM size
+* Virtual - is the host hardware virtual? yes/no
+* Disks total space - total host storage space
+* Kernel Version - Kernel version of the host operating system
+* BIOS vendor - vendor of the host BIOS
+* BIOS release date - release date of the host BIOS
+* BIOS version - version of the host BIOS
+
+Puppet::
```

Comment:
I'm not sure how to deal with this, but AFAIK group this is only there when the Puppet plugin is installed. For upstream Foreman that's true by default, for Katello and Satellite not and the user has to opt in.

Reply:
Understood. I added the note.

Reply:
Yes, this should be only present if Puppet plugin is enabled.